src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -143,3 +150,46 @@ def _describe(self, catalog: str, database: str, table: str) -> Table | None:
         except Exception as e:
             logger.error(f"Couldn't fetch information for table {full_name} : {e}")
             return None
+
```

Comment:
I think we need to introduce a new class, something like `TablesMigration` that will take `TablesCrawler` as the argument.

Our sync status check procedure should be:
- `for table in self._tables.snapshot(): ...`
- get a list of target catalogs
- `for table in self._ws.tables.list(target_catalog, ...)` to fetch the list of already migrated tables
- check if the `migrated_from` property is set for the relevant source table
- for every table, do the migration
- for every table in the `hive_metastore`, set `migrated_to={target_catalog}.{db}.{table}`
- for every table in the target catalog, set `migrated_from=hive_metastore.{db}.{table}`

Later, this `TablesMigration` class will have more stuff in it, like views migration (but that's outside of scope of this PR)